commit 73e7b152ac8a5c83d93c2ae9ae9e0e90a506d37d
Author: Stefan Saraev <stefan@saraev.ca>
Date:   Thu Mar 22 13:13:22 2012 +0200

    fix ca descr parsing for caid 4AEE
     - bulsatcom on hellas 39e, bulcrypt, NOT drecrypt

diff --git a/src/psi.c b/src/psi.c
index 130678d..116e001 100644
--- a/src/psi.c
+++ b/src/psi.c
@@ -307,8 +307,10 @@ psi_desc_ca(service_t *t, const uint8_t *buffer, int size)
     }
     break;
   case 0x4a00://DRECrypt
-    provid = size < 4 ? 0 : buffer[4];
-    break;
+    if (caid != 0x4aee) { // Bulcrypt
+      provid = size < 4 ? 0 : buffer[4];
+      break;
+    }
   default:
     provid = 0;
     break;
